zcash_client_sqlite/src/testing.rs

```diff
@@ -1099,24 +1152,22 @@ fn fake_compact_block_spending<P: consensus::Parameters, Fvk: TestFvk>(
     let mut rng = OsRng;
```

Comment:
The `PaymentAddress` argument of this function should probably become an `Address`.

Reply:
That change is out of scope for what I was able to finish by 1am, because it requires a completely separate generalization, not `TestFvk`

Reply:
Now that it's 1pm instead, indeed this particular generalization is trivial now that I've done the `TestFvk` part. Added it to the PR.